### PR TITLE
Add support for encoding shortcodes.

### PR DESCRIPTION
**Pull request by saifsultanc**
## Context

⛑️ Ticket(s): https://secure.helpscout.net/conversation/2292239606/51747?folderId=3808239

## Summary

Add support for encoding shortcodes.

## Checklist

- [ ] Updated customer telling them that a fix/addition is in the works.
- [ ] Added/Improved Cypress tests or a note under _Summary_ why tests are not included in the PR.
- [ ] Added a link to this PR in the Help Scout ticket(s) in the form of a note
- [ ] Sent a packed build

### CODE REVIEW

File: class-gwiz-gf-openai.php [hunk 1]
```diff
@@ -1120,6 +1120,7 @@ public function process_endpoint_chat_completions( $feed, $entry, $form ) {
 
 		// Parse the merge tags in the message.
 		$message = GFCommon::replace_variables( $message, $form, $entry, false, false, false, 'text' );
+		$message = GFCommon::encode_shortcodes( $message );
```
Comment (claygriffiths) [inline class-gwiz-gf-openai.php:1123]: It sounds like `do_shortcode()` is what's needed here.

**Comment by claygriffiths** [comment]
Gonna close this as I think it's better as a snippet, at least for the time being, until we see more demand for this.

File: class-gwiz-gf-openai.php (hunk 1 above)
Comment (saifsultanc) [reply class-gwiz-gf-openai.php:1123]: @claygriffiths yeah sorry. I dont know what I was thinking.

Comment (claygriffiths) [reply class-gwiz-gf-openai.php:1123]: 😂 All good!

**Pull request closed without merging**